source/crud/write-operations/modify.txt

```diff
@@ -256,3 +279,56 @@ documents match.
     For more information about constraints on unique indexes,
     see :manual:`Unique Indexes </core/index-unique/>` in the
     {+mdb-server+} manual.
+
+ReplaceOne Example: Full File
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/crud/example-intro.rst
+
+The following code is a complete, standalone file that performs a replace one operation. 
+
+.. io-code-block::
+
+   .. input:: /includes/crud/ReplaceOne.java
+      :language: java
+      :dedent:
+
+   .. output:: 
+      :language: none
+      :visible: false
+
+      UpdateOne modified document count: 1
+      Upserted ID: null
+
+      UpdateMany modified document count: 242
+
+Additional Information
+----------------------
+
+API Documentation
+~~~~~~~~~~~~~~~~~
+
+For more information about the methods and classes used to modify documents, see the following API cocumentation:
+
+- `updateOne() <{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#updateOne(org.bson.conversions.Bson,java.util.List,com.mongodb.client.model.UpdateOptions)>`__
+- `updateMany() <{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#updateMany(org.bson.conversions.Bson,java.util.List,com.mongodb.client.model.UpdateOptions)>`__
+- `UpdateOptions <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/UpdateOptions.html>`__
+- `replaceOne() <{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#replaceOne(org.bson.conversions.Bson,TDocument)>`__
+- `ReplaceOptions <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/ReplaceOptions.html?is-external=true>`__
+- `UpdateResult <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/result/UpdateResult.html>`__
+ 
+For more information about the methods used in the examples on this
```

Comment:
Q: is there a difference between this list of API documentation and the one right before? I think it makes sense to combine them into one